schema/schema.cql

```diff
@@ -24,3 +24,9 @@ create table metric_name_set (
   metric_names set<varchar>,
   primary key (shard)
 );
+
+-- graphite_names
+create table graphite_names (
```

Comment:
it becomes easier later when we say, fetch data partially. it becomes much easier to work with partial data if it is explicitly sharded (I abused that fact several times while working with Blueflood).